modules/kubernetes/eks-core/variables.tf

```diff
@@ -142,7 +142,7 @@ variable "cert_manager_config" {
   description = "Cert Manager configuration"
   type = object({
     notification_email = string
-    dns_zone           = string
+    dns_zones          = list(string)
```

Comment:
Should we do the work of changing this variable to `dns_zones` vs `dns_zone` or not?
If so we should do the same in AKS.

Reply:
Changed back to using dns_zone instead. Might be ugly but we are consistent. We can change this in a future change if we want.